install_config/storage_examples/binding_pv_by_label.adoc

```diff
@@ -15,19 +15,19 @@ toc::[]
 == Overview
 This topic provides an end-to-end example for binding
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volume-claims[persistent
-volume claims] (PVCs) to
+volume claims] (Persistent Volume Claims, PVCs) to
 xref:../../architecture/additional_concepts/storage.adoc#persistent-volumes[persistent
-volumes] (PVs) by defining labels in the PV and matching selectors in the PVC.
+volumes] (Persistent Volumes, PVs), by defining labels in the PV and matching selectors in the PVC.
```

Comment:
You're right. I missed that full name as the link.  I see this pr is merged. I can create a new pr for this one change or we can try to catch it the next time the doc is changed. Preference?